app/assets/stylesheets/application.scss

```diff
@@ -52,6 +60,78 @@ body {
   padding: 0.25rem 1rem;
 }
 
+
+.link-underlined {
+  transition: all .2s ease-in-out;
+  position: relative;
+  padding: 0;
+  margin: 0;
+}
+
+.link-underlined::after {
+  content: "";
+  position: absolute;
+  bottom: -5px;
+  left: 0;
+  width: 0;
+  height: 3px;
+  margin: 5px 0 0;
+  border-radius: 3px;
+  transition-duration: .3s;
+  opacity: 0;
+  transition: all .2s ease-in-out;
+}
+
+
+.link-underlined-yellow::after {
+    background-color: #DEB841;
+}
+
+.link-underlined:hover:after {
+  width: 100%;
+  opacity: 1;
+}
+
+
+.btn {
+  margin: 0px;
+  color: #0D3B66;
+}
+
+.btn-flat.create-event {
+  padding: 8px 24px;
+  border-radius: 4px;
+  background: #02c7cb;
+  transition: background 0.3s ease;
+  margin: 0;
```

Comment:
It is a minor formatting issue: Sometimes we write "0px" and sometimes just "0". I think it would be better to have one option only in the document. But this is "high-level moaning".

Reply:
I'm a fan of consistency, too. Added px 👍